src/main/java/it/pagopa/ecommerce/payment/methods/controller/v1/PaymentMethodsController.java

```diff
@@ -289,4 +296,53 @@ private Mono<String> getAuthenticationToken(ServerWebExchange exchange) {
                         .map(header -> header.substring("Bearer ".length()))
         );
     }
+
+    @Warmup
+    public void getAllPaymentMethodsWarmupMethod() {
+        WebClient
+                .create()
+                .get()
+                .uri("http://localhost:8080/payment-methods")
+                .header(X_CLIENT_ID, PaymentMethodRequestDto.ClientIdEnum.CHECKOUT.toString())
+                .retrieve()
+                .bodyToMono(PaymentMethodsResponseDto.class)
+                .block(Duration.ofSeconds(30));
+    }
+
+    @Warmup
+    public void calculateFeesWarmupMethod() {
+        CalculateFeeRequestDto request = new CalculateFeeRequestDto()
+                .touchpoint("touchpoint1")
+                .paymentAmount(1L)
+                .primaryCreditorInstitution("77777777777")
+                .transferList(Collections.emptyList())
+                .isAllCCP(false);
+        WebClient
+                .create()
+                .post()
+                .uri(
+                        "http://localhost:8080/payment-methods/{id}/fees",
+                        UUID.randomUUID().toString()
+                )
+                .bodyValue(request)
+                .header(X_CLIENT_ID, PaymentMethodRequestDto.ClientIdEnum.CHECKOUT.toString())
+                .retrieve()
+                .toBodilessEntity()
+                .block(Duration.ofSeconds(30));
+    }
+
+    @Warmup
+    public void createSessionWarmupMethod() {
+        WebClient
+                .create()
+                .post()
+                .uri(
+                        "http://localhost:8080/payment-methods/{id}/sessions",
+                        UUID.randomUUID().toString()
```

Comment:
please add a new env variable from which read the payment method id to be used during this warmup api call in order to make warmup request to be processed successfully.
Payment method ids to be used are (based on the env):

- DEV -> e7058cac-5e1a-4002-8994-5bab31e9f385
- UAT -> 378d0b4f-8b69-46b0-8215-07785fe1aad4
- PROD -> 83b5e0c7-2d35-44f4-9e48-a168b84433a1

Reply:
Thanks for the ids. Done.